### PR TITLE
Improve localization

### PR DESCRIPTION
**Pull request by friscoMad**
Added all missing strings to locales:

 * Trainer table headers
 * Pokemon table headers
 * Home ivs
 * Gym details
 * Graphs
 * Pokestops
 * IVs
 * etc

Improved localization of candies in pokedex for other languages

Improve general localization

~~Only UI changes~~ (Mostly UI changes, needed to change some aru responses to include locales)
Working on http://map.neoseo.es

The only part that has not been tested is the IV info in home and top 50 pokemon in pokedex as my instance does not encounter pokemon.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by pki0** [comment]
I like this. Thanks!
What about the "Today" and x "Days" at the bottom of each pokemon on trainer page?

**Comment by friscoMad** [comment]
@pokemonkiel That part and the stats char inside pokedex are not that easy to translate as it is done via javascript, so we should pass that translated text somehow to javascript, it is not done in any other part and I wanted to leave this PR just for the obvious and simple changes.

**Comment by Obihoernchen** [comment]
One +1 and I'll merge 👍 thx

**Comment by pokemonkiel** [comment]
Okay I'll have a look at JS Translation later today. When I can figure it out fast we can add this in this commit

**Comment by pki0** [comment]
@friscoMad I made a pull request into your fork with the changes.
In my tests it worked :)
+ Added German Translation
Just commit the PR to your branch

**Review by pki0 — CHANGES_REQUESTED** [review]
Commit my PR to your branch please

**Comment by friscoMad** [comment]
Ok, now it does include also the localization of pokemon info in trainers page thanks to @pokemonkiel 
I will not change anything else unelss requested.
Somebody should test in an instance with wild pokemon encounters to check IV in home for the mythics and the table of best 50 wild pokemon in pokedex.

**Review by pki0 — APPROVED** [review]

File: core/json/locales/DE/translations.json [hunk 1]
```diff
@@ -17,8 +18,11 @@
 "DASHBOARD_SPAWN_TOTAL": "Spawns gesamt",
 "DASHBOARD_SUBTITLE": "Die Daten werden alle <strong>15 Minuten</strong> aktualisiert",
 "DASHBOARD_TITLE": "MEHR <strong>DATEN</strong> MEEEEEHR",
+"DATE": "Datum",
+"TODAY" "Heute",
```
Comment (pki0) [inline core/json/locales/DE/translations.json]: Please add the :

File: core/json/locales/DE/translations.json [hunk 2]
```diff
@@ -33,21 +37,26 @@
...
 "MINUTES": "Minuten",
 "MONTH": "Monat",
 "MONTHS": "Monate",
+"MOVE": "Angriff",
 "MYSTIC": "Weisheit",
+"NAME": "Name",
 "NAV_DASHBOARD": "Dashboard",
 "NAV_POKEDEX": "Alle Pokémon",
 "NEVER": "Noch nie",
+"NOT_AVAILABLE" "Nicht verfügbar",
```
Comment (pki0) [inline core/json/locales/DE/translations.json]: Please add the :

**Comment by pki0** [comment]
It's working for me in forced German and English language.
@friscoMad Please correct my two mistakes in German translation :)

**Review by pki0 — CHANGES_REQUESTED** [review]

**Review by pki0 — APPROVED** [review]
Works now. Thanks!
Can be merged now in my opinion

**Comment by friscoMad** [comment]
Ok sorry for the mess, I tried to rebase but I just messed everything, it is ready to be merged again.

**Comment by Obihoernchen** [comment]
Merge ETA: next week. Sorry for the delay.

**Review by pki0 — CHANGES_REQUESTED** [review]
## Working:
- Trainer Pokemon tables IV bars
image

## Not working/Missing:
- home.page IV Bars (You missed aru.php)  
image
https://github.com/brusselopole/Worldopole/blob/master/core/process/aru.php#L205
- Live Pokemon popups IV bars
image
https://github.com/brusselopole/Worldopole/blob/master/core/js/pokemon.maps.js#L315

- Trainer graph on trainer page
bildschirmfoto 2017-03-28 um 11 56 18
https://github.com/brusselopole/Worldopole/blob/master/core/js/trainer.graph.js.php#L87

- Gym page popups IV bars
image
https://github.com/brusselopole/Worldopole/blob/master/core/process/aru.php#L527

- Pokemon.page Top tables: A /D /Sta   , non translated abbreviations
bildschirmfoto 2017-03-28 um 11 54 42
https://github.com/brusselopole/Worldopole/blob/master/pages/pokemon.page.php#L298
https://github.com/brusselopole/Worldopole/blob/master/pages/pokemon.page.php#L340

- Pokemon.page Polarchart
image
https://github.com/brusselopole/Worldopole/blob/master/core/js/pokemon.graph.js.php#L99

-Pokemon.page FB/Twitter Buttons:
image
https://github.com/brusselopole/Worldopole/blob/master/pages/pokemon.page.php#L34

- Footer "Made in Brussels" (wish from Discord)
image

https://github.com/brusselopole/Worldopole/blob/master/index.php#L131




**This was a quick testing.**

I'll add German translation after merge or Obi can do it. So we don't mess up shithub

**Comment by RamiroWincor** [comment]
Sorry for the delay I think I can get it updated this weekend, I needed to do some work on RocketMap first so I could stabilize my server.

**Comment by friscoMad** [comment]
Ok I did have a busy week trying to fix some things in my RM instance, I will work on this during this weekend to have it working asap.

**Comment by friscoMad** [comment]
I think it is done, at least until someone finds something else that needs to translate.
I did not translate default gym UI (the ugly one) in gym map on purpouse. It can be done when it is fixed or removed.

As stated on the description my instance does not have encounters so please check that part before merging.

File: pages/trainer.page.php [hunk 3]
```diff
@@ -52,15 +52,15 @@
...
+		  <button type="submit" class="btn btn-primary"><?= $locales->SEARCH ?></button>
 		</form>
 	</div>
 </div>
 </header>
 
 <div class="row area" id="trainersGraph">
 	<div class="col-md-12 text-center">
-		<h2 class="sub-title"><?= $locales->TRAINER_GRAPH ?></h2>
+		<h2 class="sub-title"><?= $locales->TRAINERS_GRAPH ?></h2>
```
Comment (Obihoernchen) [inline pages/trainer.page.php:63]: Please change the key to TRAINER**S**_GRAPH in translations.json for all languages.

File: pages/trainer.page.php [hunk 4]
```diff
@@ -76,19 +76,19 @@
...
+				<th><?= $locales->TRAINERS_TABLE_LAST_SEEN ?></th>
 			</tr>
 			</thead>
 			<tbody id="trainersContainer">
 
 			</tbody>
 			<tfoot>
 				<tr class="loadMore text-center">
-					<td colspan="6"><button id="loadMoreButton" class="btn btn-default hidden"><?= $locales->TRAINER_LOAD_MORE ?></button></td>
+					<td colspan="6"><button id="loadMoreButton" class="btn btn-default hidden"><?= $locales->TRAINERS_LOAD_MORE ?></button></td>
```
Comment (Obihoernchen) [inline pages/trainer.page.php:91]: Please change the key to TRAINER**S**_LOAD_MORE in translations.json for all languages.

**Review by Obihoernchen — CHANGES_REQUESTED** [review]

**Comment by Obihoernchen** [comment]
See comments above. Everything else seems to be fine I think 👍

**Comment by friscoMad** [comment]
@Obihoernchen My bad, I have updated the translations missing.

**Review by Obihoernchen — APPROVED** [review]

**Comment by Obihoernchen** [comment]
One more +1 and I'll merge.

**Comment by Obihoernchen** [comment]
pls rebase <3

**Comment by friscoMad** [comment]
Done, somebody pls +1 so it can merge or it will be a rebase nightmare, this PR touches almost all files 😭

**Review by Obihoernchen — APPROVED** [review]

**Comment by Obihoernchen** [comment]
I'll not merge anything else before this.

File: core/json/locales/EN/translations.json [hunk 5]
```diff
@@ -33,21 +38,29 @@
...
 "GYMS_TITLE": "The <strong>fight</strong> map<br><small>Follow the conquest of your team</small>",
+"GYMS_NO_DEFENDER": "No Defender yet",
 "HOME": "Home",
 "HOUR": "hour",
 "HOURS": "hours",
 "INCITY": "in %s",
 "INSTINCT": "Instinct",
+"IV_ATTACK": "IV Attack",
+"IV_DEFENSE": "IV Defense",
+"IV_STAMINA": "IV Stamina",
```
Comment (Obihoernchen) [inline core/json/locales/EN/translations.json]: Please use `ATTACK`, `DEFENSE` and `STAMINA` only and just prepend `"IV "`. We don't have to translate this twice.

**Review by Obihoernchen — APPROVED** [review]

**Review by Obihoernchen — APPROVED** [review]

**Pull request merged**